notebooks/checks/performance/regression_systematic_error.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
Some things that can improve the graph:

1. Add some text above graph explaining what are we trying to see (e.g. " non-zero mean of the error distribution indicated the pretense of systematic error in model predictions") 
2. Add some text to explain what are we seeing (e.g. "box plot of the distribution of model error")
2. Make it so that the mean is not 0, so we can see what it's about